wiki/Contests/oMWC/2/en.md

```diff
@@ -0,0 +1,158 @@
+
+4. **What is a “Team Captain”?**
+
+   The Team Captains, also called Captains are chosen by the mappers of their country through a vote on the website. They will be the ones in charge to communicate with the organizers and their team in order to let us know about team-related things. The requirements to become Team Captain is to have at least one ranked guest difficulty, in order to show your mapping experience.
+
+5. **What if my country doesn’t have a Team Captain?**
+
+   You’re free to apply as one through the website if you meet the criterias! Mappers of your country will vote for the people they want to have as their country’s Team Captain. If nobody applied to be a Team Captain for your country once the deadline is over, contest staff will pick one for the team. The country you will be allowed to participate with will be based on your profile's flag. No country merge will be allowed to avoid management complications.
+
+6. **Can I have my own team for my country even if it already has one?**
+
+   No. Each country will only have one Team Captain to represent it.
+
+7. **How do we know who will face each other? Will it be randomized?**
+
+   No, it won’t be randomized. During the Qualifier round, only the best 16 teams will move to the clash rounds. The first team will face the sixteenth, the second will face the fifteenth, the third will face the fourteenth, etc… So make sure to give all your energy during the Qualifiers stage!
+
+8. **Do I have to join the Discord server to participate?**
+
+   Only the Team Ambassador is required to join the Discord server. The team members however are free to join or not. Although not joining will result in you missing out on important announcements and a private country text channel which will be ONLY available on the discord server.
+
+9. **Can I join the Discord server even if I don’t want to participate in the contest?**
+
+   Yes. There is a public link at the top of the forum post. Upon joining, you’ll be in the #arrival channel, where you will have to post your osu! profile link, and then a member of the staff or a moderator will accept you.
+
+10. **“At least 2 mappers will have to collaborate on each song” - Are we able to work with my full team on each map?**
+
+    Yes! We set this rule to make sure the main spirit of the contest about collaborations remains, while not restricting people with their team management. If you want to work on a full team on each stage, you’re encouraged to do so!
+
+11. **I want to vote, but I don’t have any mapping experience, am I allowed to do so?**
+
+    Sadly, no you can’t. You must have at least one ranked guest difficulty in osu!standard. If you meet these criterias, you’ll be able to vote for the potential Team Captains based on your country flag, which means you can’t vote for Russian Team Captain if you’re from the United Kingdom.
```

Comment:
I think `osu!standard` is a frowned-upon expression? it does make things clear however

```suggestion
    Sadly, no you can’t. You must have at least one ranked guest difficulty in osu!standard. If you meet these criteria, you’ll be able to vote for the potential Team Captains based on your country flag, which means you can’t vote for the Russian Team Captain if you’re from the United Kingdom.
```